bundle/edu.gemini.ags.servlet/src/main/java/edu/gemini/ags/servlet/ToContext.java

```diff
@@ -269,20 +229,14 @@ private InstAltair getAltair(HttpServletRequest req) throws RequestException {
      * @return
      */
     private static AltairParams.Mode modeFromAgsServletParameter(String paramName) {
```

Comment:
Yes, I agree.  I just switched to an easier to read `switch` statement in passing but a more extensive refactoring would probably be in order at some point.